apps/web/src/design-system/cards/CardTile.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled';
+import { colors, shadows } from '../config';
+
+export const CardTile = styled.button`
```

Comment:
moved this one to components, and it's reused between the workflows page and integrations list page